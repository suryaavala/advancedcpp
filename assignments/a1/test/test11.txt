2
2 repeat
sqrt
endrepeat