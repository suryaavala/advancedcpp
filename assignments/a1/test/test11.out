sqrt 2 = 1
sqrt 1 = 1
